samples/cellular/location/README.rst

```diff
@@ -70,16 +70,24 @@ Building and running
 
 .. include:: /includes/build_and_run_ns.txt
 
-nRF9160 DK with nRF7002 EK Wi-Fi support
-========================================
+nRF91 Series DK with nRF7002 EK Wi-Fi support
+=============================================
 
-To build the sample with nRF9160 DK and nRF7002 EK Wi-Fi support, use the ``-DSHIELD=nrf7002ek``, ``-DDTC_OVERLAY_FILE=nrf9160dk_with_nrf7002ek.overlay`` and  ``-DOVERLAY_CONFIG=overlay-nrf7002ek-wifi-scan-only.conf`` options.
+To build the sample with nRF91 Series DK and nRF7002 EK Wi-Fi support, use the following commands:
 
-For example:
+.. tabs::
 
-.. code-block:: console
+   .. group-tab:: nRF9161 DK
+
+      .. code-block:: console
+
+          west build -p -b nrf9161dk_nrf9161_ns -- -DOVERLAY_CONFIG=overlay-nrf7002ek-wifi-scan-only.conf -DSHIELD=nrf7002ek
```

Comment:
The `nrf9160dk_with_nrf7002ek.overlay` overlay file is needed also with nRF9161 DK. The name of the overlay is misleading as it's used with both DKs, we'll update it.
```suggestion
          west build -p -b nrf9161dk_nrf9161_ns -- -DSHIELD=nrf7002ek -DDTC_OVERLAY_FILE=nrf9160dk_with_nrf7002ek.overlay -DOVERLAY_CONFIG=overlay-nrf7002ek-wifi-scan-only.conf
```

Reply:
Created PR #13168 which renames the nRF7002 EK overlays in `location` and `modem_shell` samples.

Reply:
I will update these changes in this PR.